lifechanger/src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
 <app-header></app-header>
-<app-footer></app-footer>
+<app-hero></app-hero>
```

Comment:
Here it's not rigth set uo for routing. You should take a look one more time how Angular routing work. You should think about usage of <router-outlet>

Reply:
![image](https://github.com/TEZv/Become_PRO_with_Angular/assets/34271977/ffc46610-55bf-46e3-a7cf-97fa606207b0)
![image](https://github.com/TEZv/Become_PRO_with_Angular/assets/34271977/843b927a-87a0-4165-957e-35b210495a62)
![image](https://github.com/TEZv/Become_PRO_with_Angular/assets/34271977/8ec04dff-ffaa-4790-9b92-056d61e33928)

Reply:
Dear Mentor, thank you for reviewing the code.
I repaired the **routing** according to the provided images and remarks.
Thank you